src/Core/Models/Api/Request/Organizations/OrganizationUserRequestModels.cs

```diff
@@ -96,4 +96,10 @@ public class OrganizationUserBulkReinviteRequestModel
         [Required]
         public IEnumerable<Guid> Ids { get; set; }
     }
+    
+    public class OrganizationUserBulkRemoveRequestModel
+    {
+        [Required]
+        public IEnumerable<Guid> Ids { get; set; }
+    }
```

Comment:
Are we expecting these bulk org user actions to have different parameters per action? If not, it might be worth renaming OrganizationUserBulkReinviteRequest to something more generic like OrganizationUserBulkActionRequest and using that were applicable instead of having separate models.